Doc/library/functools.rst

```diff
@@ -373,20 +373,24 @@ The :mod:`functools` module defines the following functions:
    only one positional argument is provided, while there are two placeholders
    in :ref:`partial object <partial-objects>`.
 
-   When successively using :func:`partial` existing :data:`!Placeholder`
-   sentinels are filled first. A place for positional argument is retained
-   when :data:`!Placeholder` sentinel is replaced with a new one:
+   Successive :func:`partial` applications fill :data:`!Placeholder` sentinels
+   of the input :func:`partial` objects with new positional arguments.
+   A place for positional argument can be retained by inserting new
+   :data:`!Placeholder` sentinel to the place held by previous :data:`!Placeholder`:
 
       >>> from functools import partial, Placeholder as _
-      >>> count = partial(print, _, _, _, 4)
-      >>> count = partial(count, _, _, 3)
-      >>> count = partial(count, _, 2)
-      >>> count = partial(count, _, 5)   # 5 is appended after 4
-      >>> count(1)
+      >>> show5 = partial(print, _, _, _, 4)
+      >>> show5 = partial(show5, _, _, 3)
+      >>> show5 = partial(show5, _, 2)
+      >>> show5 = partial(show5, _, 5)   # 5 is appended after 4
```

Comment:
I am not sure that it is worth to keep such weird example, but I would use different names:
```
      >>> show4 = partial(print, _, _, _, 4)
      >>> show3 = partial(show4, _, _, 3)
      >>> show2 = partial(show3, _, 2)
      >>> show5 = partial(show2, _, 5)   # 5 is appended after 4
```
Well, it still does not make sense.

If you want more realistic examples, I can propose something like:
```
>>> remove = partial(str.replace, _, _, '')
>>> remove('Hello, world', 'l')
'Heo, word'
>>> remove_l = partial(remove, _, 'l')
>>> remove_l('Hello, world')
'Heo, word'
>>> remove_first_l = partial(remove_l, _, 1)
>>> remove_first_l('Hello, world')
'Helo, world'
```